pkg/driver/driver.go

```diff
@@ -178,10 +178,10 @@ func Start(ctx context.Context, driverName string, kubeClient kubernetes.Interfa
 }
 
 func (np *NetworkDriver) Stop() {
+	// Stop NRI Plugin and wait for it to stop.
 	np.nriPlugin.Stop()
-	np.draPlugin.Stop()
-}
+	np.nriPlugin.Wait()
```

Comment:
Im not familiar with this method but the nri library has some surprising behavior

Reply:
I prefer to not add it, as commented above NRI library has some unexpected behaviors and I do not want us to be the first in finding those bugs, specially since we do not really need to wait for NRI to stop

Reply:
What happens if we don't stop NRI and try and start a new DraNet plugin, i.e on a failure. Could that cause issues if we don't wait?